pages/common/pyenv.md

```diff
@@ -0,0 +1,27 @@
+# pyenv  
+
+> Switch between multiple versions of Python easily.
+
+- List all available commands:
+
+`pyenv commands`
+
+- List all Python versions known to pyenv:
+
+`pyenv versions`
+
+- Install a Python version:
```

Comment:
Is this installation global, pyenv-specific, or something else? It would be nice to clarify that, just to be sure.